src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts

```diff
@@ -5,7 +5,7 @@ namespace Providers.OSUI.RangeSlider.NoUISlider.Utils {
 		const _noUiSliderConfigs = providerConfigs;
 
 		// Check if the range key is empty. If true, remove from the array
-		if (_noUiSliderConfigs.range.length <= 0) {
+		if (_noUiSliderConfigs.range !== undefined && _noUiSliderConfigs.range.length <= 0) {
```

Comment:
You could simply do ` if (_noUiSliderConfigs.range?.length <= 0) {}`

Reply:
Indeed! at the end of compilation it will be the same... But more clean on our side yes! Thank you ;)

Reply:
Done!